projects/train/train/data/waveforms/generator/cbc.py

```diff
@@ -158,6 +158,16 @@ def frequency_domain_strain(self, **parameters):
         self.approximant.to(device)
         return self.waveform(freqs[self.freq_mask], **parameters)
 
+    def time_and_frequency_domain_strain(self, **parameters):
+        hc, hp = self.waveform(self.frequencies[self.freq_mask], **parameters)
```

Comment:
Since you use the `time` prefix for the time domain waveform, could you also use a `freq` prefix for the frequency domain waveform?